### PR TITLE
Publish release candidate tags

### PR DESCRIPTION
**Pull request by punya**
This change is [Reviewable](https://reviewable.io/reviews/palantir/gradle-java-distribution/149)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ deployment:
   #  commands:
   #    - ./gradlew -i artifactoryPublish
   release:
-    tag: /[0-9]+(\.[0-9]+)+/
+    tag: /[0-9]+(\.[0-9]+)+(-rc[0-9])?/
```
Comment (bmoylan) [inline circle.yml]: I think this is fine, but want to make sure it's intentional to _not_ match `1.0-rc` (no number) or `1.0-rc10` (>1 digit)

Comment (punya) [reply circle.yml]: The former is intentional, the latter is just to do the minimal thing that will let me make progress. Happy to use something more extensive like `/[0-9]+(\.[0-9]+)+(-rc[0-9]+)?(-alpha[0-9]+)?/`.

Comment (punya) [reply circle.yml]: (That one's cribbed from http-remoting FWIW.)

**Comment by punya** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[circle.yml, line 30 at r1](https://reviewable.io:443/reviews/palantir/gradle-java-distribution/149#-KXwKGFS3jm3qglk5IHJ-r1-30:-KXwN7GBb32AUTPP8G9_:b-896fix) ([raw file](https://github.com/palantir/gradle-java-distribution/blob/7db0ad0ea95a5cba22c6fb9975e0e9c1ed0cc7bc/circle.yml#L30)):*
<details><summary><i>Previously, punya (Punyashloka Biswal) wrote…</i></summary><blockquote>

(That one's cribbed from http-remoting FWIW.)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/gradle-java-distribution/149)*

**Comment by bmoylan** [comment]
👍

**Comment by bmoylan** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/palantir/gradle-java-distribution/149)*

**Pull request merged**